src/variations.js

```diff
@@ -0,0 +1,320 @@
+window.VARIATIONS = {
+  "TPL0": {
+    weight: 1,
+    prefs: {
+      // Always turn on Content Blocking.
+      "browser.contentblocking.enabled": true,
+      "browser.contentblocking.ui.enabled": true,
+
+      // Make sure we're not affected by the Symantec distrust.
+      "security.pki.distrust_ca_policy": 1,
+
+      "browser.fastblock.enabled": false,
+      "privacy.trackingprotection.enabled": true,
+      "network.http.tailing.enabled": false,
+
+      // Disable FastBlock UI.
+      "browser.contentblocking.fastblock.ui.enabled": false,
+      "browser.contentblocking.fastblock.control-center.ui.enabled": false,
+    },
+  },
+
+  "TPL1": {
+    weight: 1,
+    prefs: {
+      // Always turn on Content Blocking.
+      "browser.contentblocking.enabled": true,
+      "browser.contentblocking.ui.enabled": true,
+
+      // Make sure we're not affected by the Symantec distrust.
+      "security.pki.distrust_ca_policy": 1,
+
+      "browser.fastblock.enabled": false,
+      "privacy.trackingprotection.enabled": true,
+      "network.http.tailing.enabled": false,
+      // TODO should this be set?
+      //"urlclassifier.trackingTable": "",
+      //"urlclassifier.trackingWhitelistTable": "",
+
+      // Disable FastBlock UI.
+      "browser.contentblocking.fastblock.ui.enabled": false,
+      "browser.contentblocking.fastblock.control-center.ui.enabled": false,
+    },
+  },
+
+  "TPL2": {
+    weight: 1,
+    prefs: {
+      // Always turn on Content Blocking.
+      "browser.contentblocking.enabled": true,
+      "browser.contentblocking.ui.enabled": true,
+
+      // Make sure we're not affected by the Symantec distrust.
+      "security.pki.distrust_ca_policy": 1,
+
+      "browser.fastblock.enabled": false,
+      "privacy.trackingprotection.enabled": true,
+      "network.http.tailing.enabled": false,
+      // TODO should this be set?
+      //"urlclassifier.trackingTable": "",
+      //"urlclassifier.trackingWhitelistTable": "",
+
+      // Disable FastBlock UI.
+      "browser.contentblocking.fastblock.ui.enabled": false,
+      "browser.contentblocking.fastblock.control-center.ui.enabled": false,
+    },
+  },
+
+  "TPL3": {
+    weight: 1,
+    prefs: {
+      // Always turn on Content Blocking.
+      "browser.contentblocking.enabled": true,
+      "browser.contentblocking.ui.enabled": true,
+
+      // Make sure we're not affected by the Symantec distrust.
+      "security.pki.distrust_ca_policy": 1,
+
+      "browser.fastblock.enabled": false,
+      "privacy.trackingprotection.enabled": true,
+      "network.http.tailing.enabled": false,
+      // TODO should this be set?
+      //"urlclassifier.trackingTable": "",
+      //"urlclassifier.trackingWhitelistTable": "",
+
+      // Disable FastBlock UI.
+      "browser.contentblocking.fastblock.ui.enabled": false,
+      "browser.contentblocking.fastblock.control-center.ui.enabled": false,
+    },
+  },
+
+  "FB2L0": {
```

Comment:
this one is missing `urlclassifier.trackingAnnotationTable` and `urlclassifier.trackingAnnotationWhitelistTable`

though I'm not sure what it should be set to for list 0 - as it seemed that `fastblock3` was actually the default

Reply:
For tailing, considering that it's already the default, I thought it was unnecessary to force on, but I guess it doesn't hurt to do it either.

Reply:
And I think it's better to not enable pref globally anywhere, so that we have an explicit list of which prefs will be modified in `variations.js`